Resources/public/js/alloyeditor/buttons/embed.jsx

```diff
@@ -51,6 +63,23 @@ YUI.add('ez-alloyeditor-button-embed', function (Y) {
         },
 
         /**
+         * Checks if the command is disabled in the current selection.
+         *
+         * @method isDisabled
+         * @return {Boolean} True if the command is disabled, false otherwise.
+         */
+        isDisabled: function () {
+            var path = this.props.editor.get('nativeEditor').elementPath();
+
+            // http://docs.ckeditor.com/#!/api/CKEDITOR.dom.elementPath
+            // There is also isContextFor( tag ), so if there is a way to specify where embeds
+            // are valid that would potentially be cleaner
+            this.state.isDisabled = path && path.contains('table', true) !== null;
```

Comment:
> also as you wrote in the comment above, the is disabled logic should not be here but rather in the embed plugin.

I wasn't saying embed plugin, but maybe indeed that is where logic for isContextFor would have to live. But for now I'll cleanup the comment.